src/components/ContactForm/index.jsx

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable react/no-unknown-property */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useState } from "react";
+import "./index.css";
+import { Row, Col } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faYoutube,
+  faFacebook,
+  faTwitter,
+  faInstagram,
+} from "@fortawesome/free-brands-svg-icons";
+import contactUsChannelsImg from "./contactUsChannelsImg.svg";
+import PiggyImg from "./PiggyImg.svg";
+import db from "../../firebaseConfig";
+
+export default function ContactForm() {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [message, setMessage] = useState("");
+  const { t } = useTranslation();
+
+  const messageRecieve = (e) => {
+    e.preventDefault();
+    db.collection("messages").doc(`${name}, ${Date()}`).set({
+      fullName: name,
+      email,
+      message,
+    });
+
+    setName("");
+    setEmail("");
+    setMessage("");
+    // eslint-disable-next-line no-alert
+    alert("Your message was successfully submitted");
```

Comment:
Alert is a text and it should be coming from i18next

Reply:
Fixed. PTAL